packages/components/src/components/post-accordion-item/post-accordion-item.scss

```diff
@@ -1,22 +1,149 @@
-@use '@swisspost/design-system-styles/components/accordion';
-@use '@swisspost/design-system-styles/mixins/accordion' as accordion-mx;
+@use '@swisspost/design-system-styles/mixins/button' as button-mx;
+@use '@swisspost/design-system-styles/mixins/focus' as focus-mx;
+@use '@swisspost/design-system-styles/mixins/icons' as icon-mx;
+@use '@swisspost/design-system-styles/mixins/utilities' as utility-mx;
+
+@use '@swisspost/design-system-styles/tokens/components';
+@use '@swisspost/design-system-styles/tokens/elements';
+@use '@swisspost/design-system-styles/tokens/helpers';
+@use '@swisspost/design-system-styles/functions/tokens';
+@use '@swisspost/design-system-styles/variables/components/accordion';
+
+tokens.$default-map: components.$post-accordion;
 
 :host {
-  display: block;
+  &::after {
+    display: block;
+    content: '';
+    margin-top: calc(tokens.get('accordion-border-bottom-width') * -1);
+    border-bottom: tokens.get('accordion-border-bottom-width')
+      tokens.get('accordion-border-bottom-style') tokens.get('accordion-enabled-border');
+  }
+}
+
+:host(:hover)::after {
+  border-color: tokens.get('accordion-hover-border');
+}
+
+.accordion-header {
+  color: tokens.get('accordion-enabled-fg');
+  font-size: tokens.get('accordion-header-font-size');
+  margin: 0;
+  line-height: tokens.get('heading-line-height', elements.$post-heading);
```

Comment:
This will be inherited from the heading styles

Reply:
I thought so as well but it is not since the accordion-header is in the shadow DOM and thus does not inherit the heading styles.